runner/copy_test.go

```diff
@@ -388,8 +388,12 @@ func (b BrokenNumRowsOfflineStore) GetTrainingSet(id provider.ResourceID) (provi
 	return nil, nil
 }
 
-func (b BrokenNumRowsOfflineStore) GetTrainingSetTestSplit(id provider.ResourceID, testSize float32, shuffle bool, randomState int) (provider.TrainingSetIterator, provider.TrainingSetIterator, func() error, error) {
-	return nil, nil, nil, nil
+func (b BrokenNumRowsOfflineStore) CreateTrainTestSplit(def provider.TrainTestSplitDef) (func() error, error) {
+	return nil, fmt.Errorf("not Implemented")
```

Comment:
`fferr` and some detail

Reply:
i'll fix these in another PR lol, these errors aren't too urgent